workitem/list_type_blackbox_test.go

```diff
@@ -186,3 +186,117 @@ func TestListType_SetDefaultValue(t *testing.T) {
 		})
 	}
 }
+
+func TestListType_ConvertToStringSlice(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	tests := []struct {
+		name           string
+		list           ListType
+		defVal         interface{}
+		expectedOutput []string
+		wantErr        bool
+	}{
+		{"convert empty value",
+			ListType{
+				SimpleType:    SimpleType{Kind: KindList},
+				ComponentType: SimpleType{Kind: KindString},
+			},
+			[]interface{}{},
```

Comment:
You're only testing for an empty interface array here. Can you please also test what happens when you put in `nil`?

Reply:
Thanks